constants/Garden.json

```
@@ -1229,6 +1229,18 @@
     "BIOFUEL": 3000,
     "VOLTA": 10000,
     "OIL_BARREL": 10000
+  },
+  "max_personal_best": {
+    "CACTUS": 2000000.0,
+    "CARROT": 3000000.0,
+    "COCOA_BEAN": 3000000.0,
```

```suggestion
    "COCOA_BEANS": 3000000.0,
```
`CropType` uses `COCOA_BEANS`.